src/app/external-cluster-wizard/steps/summary/component.ts

```diff
@@ -0,0 +1,40 @@
+// Copyright 2022 The Kubermatic Kubernetes Platform contributors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {Component, ViewEncapsulation} from '@angular/core';
+import {ExternalClusterService} from '@core/services/external-cluster';
+import {ExternalClusterModel, ExternalClusterProvider} from '@shared/entity/external-cluster';
+import {StepRegistry} from '../../config';
+
+@Component({
+  selector: 'km-external-cluster-summary',
+  templateUrl: './template.html',
+  styleUrls: ['./style.scss'],
+  encapsulation: ViewEncapsulation.None,
```

Comment:
Sorry I missed this my previous review but why do we need `ViewEncapsulation.None` here? All styles in `styles.scss` are targeting HTML in this component so why do we need to make them global?

Reply:
This was done after a discussion with @KhizerRehan basically, I need to apply my styles to the child components, so I have wrapped all those styles inside a parent class `.km-external-cluster-summary`.

Reply:
What are those styles that will be applied to child components?

Reply:
I know that but what I meant was which styles specifically will be applied to child components?

```
.km-external-cluster-summary {
  $steps: 3;

  @mixin counter-x {
    @for $i from 1 through $steps {
      .counter-#{$i} {
        &::before {
          @include mixins.size(22px);

          align-items: center;
          border-radius: 50%;
          border-style: solid;
          border-width: 1px;
          content: '#{$i}';
          display: inline-flex;
          font-size: variables.$font-size-body;
          justify-content: center;
          margin-right: 12px;
        }
      }
    }
  }

  .km-wizard-summary {
    .header {
      font-size: variables.$font-size-subhead-lg;
      margin-bottom: 16px;
    }
  }

  .summary-with-number {
    @include counter-x;
  }

  .km-provider-logo {
    height: 30px;
    margin-top: 2px;
    max-height: 30px;
    max-width: 100px;
    width: 100px;
  }

  .step {
    padding-bottom: 24px;
  }
}

Reply:
`info-container` is present in template of this component so it should work without view encapsulation. And for `km-chips` can we move that style to `eks-cluster-summary` component since its only being used there?

Reply:
moving `km-chips` inside `km-eks-cluster-summary` is not working, as there is no view encapsulation set there.

Reply:
I have added view encapsulation to `km-eks-cluster-summary`

Reply:
Why haven't you removed `km-chips` specific styles from this component?

Reply:
@Talha-Jamil-TJ remove view encapsulation from this component.

Reply:
Styles are breaking in the `external cluster detail` section if I don't use view encapsulation @Waseem826 
<img width="1263" alt="image" src="https://user-images.githubusercontent.com/56511988/189337285-bb1c4a11-712a-4364-bf7b-3a9d80796f51.png">